geode-core/src/test/java/org/apache/geode/internal/cache/TombstoneServiceTest.java

```diff
@@ -0,0 +1,79 @@
+package org.apache.geode.internal.cache;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.concurrent.ExecutorService;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import org.apache.geode.CancelCriterion;
+import org.apache.geode.cache.DataPolicy;
+import org.apache.geode.distributed.internal.CacheTime;
+import org.apache.geode.internal.cache.versions.RegionVersionVector;
+import org.apache.geode.internal.cache.versions.VersionTag;
+
+public class TombstoneServiceTest {
+  CacheTime cacheTime = mock(CacheTime.class);
+  CachePerfStats stats = mock(CachePerfStats.class);
+  CancelCriterion cancelCriterion = mock(CancelCriterion.class);
+  ExecutorService executor = mock(ExecutorService.class);
+  RegionMap regionMap = mock(RegionMap.class);
+  RegionEntry entry = mock(RegionEntry.class);
+  DistributedRegion region = mock(DistributedRegion.class);
+  VersionTag destroyedVersion = mock(VersionTag.class);
+
+  @Before
+  public void setUp() throws Exception {}
```

Comment:
The creation and assignment of the mocks should be moved to the `setUp()` method to ensure that fresh mocks are created for each test method and hence prevent multiple tests acting on the same instance of a mock and producing unexpected results.

Reply:
Done